src/appleseed/renderer/kernel/lighting/backwardlightsampler.h

```diff
@@ -0,0 +1,260 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
+// Copyright (c) 2014-2017 Francois Beaune, The appleseedhq Organization
+// Copyright (c) 2017 Petra Gospodnetic, The appleseedhq Organization
```

Comment:
Maybe one day we'll simplify the copyright rules... But for now you need to either remove the original copyright notices, or remove the last one.